app/src/main/java/org/oppia/android/app/survey/surveyitemviewmodel/UserTypeItemsViewModel.kt

```diff
@@ -97,46 +98,36 @@ class UserTypeItemsViewModel @Inject constructor(
 
   private fun getUserTypeOptions(): ObservableArrayList<MultipleChoiceOptionContentViewModel> {
     val observableList = ObservableArrayList<MultipleChoiceOptionContentViewModel>()
-    observableList += UserTypeAnswer.values()
-      .filter { it.isValid() }
```

Comment:
if `fun isValid()` is nolonger being used in this class then it should be removed.

Reply:
I am using the isValid function in my enumfilter for conditions what to filter .